code/datums/elements/mirage_border.dm

```diff
@@ -0,0 +1,42 @@
+/**
+ * Creates a mirage effect allowing you to see around the world border, by adding the opposite side to its vis_contents.
+ */
+/datum/element/mirage_border
+
+/datum/element/mirage_border/Attach(datum/target, turf/target_turf, direction, range=world.view)
+	. = ..()
+	if(!isturf(target))
+		return ELEMENT_INCOMPATIBLE
+	if(!target_turf || !istype(target_turf) || !direction)
+		. = ELEMENT_INCOMPATIBLE
+		CRASH("[type] improperly attached with the following args: target=\[[target_turf]\], direction=\[[direction]\], range=\[[range]\]")
```

Comment:
```suggestion
		stack_trace("[type] improperly attached with the following args: target=\[[target_turf]\], direction=\[[direction]\], range=\[[range]\]")
		return ELEMENT_INCOMPATIBLE
```

this is the preferred method of triggering the runtime message while continuing the proc here.